inspirehep/modules/editor/views.py

```diff
@@ -36,6 +43,56 @@
                       url_prefix='/editor',)
 
 
+@blueprint.route('/refextract/from_text', methods=['POST'])
```

Comment:
Do you have a strong preference for `/editor/refextract/from_text` vs. `/editor/refextract/text` ?